src/main/resources/languages/french.xml

```diff
@@ -3,7 +3,7 @@
     <meta>
         <name>Français (fr_FR)</name>
         <author>https://github.com/PPAC37</author>
-        <!-- draft v0.4.1 04/02/2022 -->
+        <!-- draft v0.4.2 13/02/2022 -->
```

Comment:
It is not a draft anymore ! the date and the version are also not useful, the history of git serves this purpose.

Reply:
old habits ... with git, indeed, it is no longer useful.